lib/commands/general.js

```diff
@@ -216,6 +221,105 @@ commands.closeApp = async function () {
   await this.adb.forceStop(this.opts.appPackage);
 };
 
+commands.getDisplayDensity = async function () {
+  let out = await this.adb.shell(['getprop', 'ro.sf.lcd_density']);
+  if (out) {
+    return parseInt(out);
+  }
+  out = await this.adb.shell(['getprop', 'qemu.sf.lcd_density']);
+  if (out) {
+    return parseInt(out);
+  }
+  log.errorAndThrow('Failed to get display density property.');
+};
+
+/**
+ * Parses the given window manager Surface string to get info.
+ * @param line: To parse. This is assumed to be valid.
+ * @return: Visibility and bounds of the Surface.
+ */
+helpers.parseSurfaceLine = function (line) {
```

Comment:
Unless there is a reason for this to be on the object (like sub-classes use it?), I'd rather this be internal to this module (`function parseSurfaceLine` instead of `helpers.parseSurfaceLine`). We have altogether too many methods just dangling off the object that are not necessary.